tutorials/tutorial7/tutorial_7.rb

```diff
@@ -0,0 +1,48 @@
+# person parent class
+class Person
+  attr_accessor :name
+
+  def initialize(name)
+    @p_name = name
```

Comment:
Please give variables as easy to read. 
@p_name => @user_name  or something.....

Reply:
Thanks for correcting me bro @HtetTunLu .I updated the tutorial_7.rb on [a03283a](https://github.com/maythuzarlinn/Ruby-Training/pull/7/commits/a03283acd59ac8fd6a46b659941e2a32af0c10f5).